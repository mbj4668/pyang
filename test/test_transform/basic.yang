module basic {
  namespace "urn:basic";
  prefix basic;
  description 'module';
  container x {
    description 'container';
    leaf y {
      type string;
    }
  }
}
