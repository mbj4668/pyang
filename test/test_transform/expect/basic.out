module: basic
  +--rw x
     +--rw y?   string
